dash/frontend/src/generic-styles.scss

```diff
@@ -65,9 +67,16 @@ $classesToGenerate: (
     values: (
       auto: auto, unset: unset,
       fit: 'fit-content',
-      0: 0, 50: 50%, 75: 75%, 100: 100%,
+      0: 0, 50: 50%, 75: 75%, 97: 97%, 100: 100%,
```

Comment:
Is this value in use anymore? (I didn't see it but maybe I just missed it.)

Reply:
@becky-intelletive  We have just removed this value.